src/structures/CommandArgsOptionBuilder.js

```diff
@@ -56,10 +56,10 @@ class CommandArgsOptionBuilder {
         this.choices = 'choices' in data ? data.choises : null;
 
         /**
-         * Args
+         * Options
          * @type {Array<CommandArgsOption>}
         */
-         this.args = 'args' in data ? data.args : null;
+         this.options = 'options' in data ? data.options : null;
```

Comment:
Why `options`?

Reply:
Although in the discord api it is, as options but they are arguments. I'd bring back args, it's much more natural to people and they're arguments.